driver-db/ens160.toml

```
@@ -0,0 +1,34 @@
+manifest_version = "0.0.1"
+
+[meta]
+categories = ["Sensor::TVOC", "Sensor::eCO2"]
+datasheets = [
+    "https://www.sciosense.com/wp-content/uploads/2023/12/ENS160-Datasheet.pdf",
+]
+manufacturer = "ScioSense"
+names = ["ENS160"]
+part_numbers = ["ENS160"]
+
+kicad_symbol = ["ENS160-BGLM"]
+
+[[dev_boards]]
+connections = ["StemmaQt"]
+link = "https://www.adafruit.com/product/5606"
+manufacturer = "Adafruit"
+name = "Adafruit ENS160 MOX Gas Sensor - Sciosense CCS811 Upgrade - STEMMA QT / Qwiic"
+
+[[dev_boards]]
+connections = ["StemmaQt"]
+link = "https://www.sparkfun.com/products/20844"
+manufacturer = "Sparkfun"
+name = "SparkFun Indoor Air Quality Sensor ENS160 for Arduino Qwiic"
+
+[[dev_boards]]
+connections = ["StemmaQt"]
```

Looking at the board I think this one does not have StemmaQt, just pins, so you can leave this empty.
```suggestion
connections = []
```